LICENSE.txt

```diff
@@ -0,0 +1,2 @@
+Copyright © 2020 Justin Shaw
+All Rights Reserved.
```

Comment:
Do you think this is enough?

Reply:
I may add something about the project it's being used for. I read the links you included. Do you plan on this project being open-source? Or private?